Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -868,6 +868,13 @@ - (void)addThreadToProfileWhitelist:(TSThread *)thread
         TSGroupThread *groupThread = (TSGroupThread *)thread;
         NSData *groupId = groupThread.groupModel.groupId;
         [self addGroupIdToProfileWhitelist:groupId];
+
+        // When we add a group to the profile whitelist,
```

Comment:
comment nit: it's usually more helpful to explain why you're doing a thing rather than just restating what the code does. The code is pretty legible as is.